bookshelf/bookshelf.d.ts

```diff
@@ -177,8 +191,12 @@ declare module 'bookshelf' {
 			keys() : string[];
 			last() : T;
 			lastIndexOf(value : any, fromIndex? : number) : number;
-			map(predicate? : Lodash.ListIterator<T, boolean>|Lodash.DictionaryIterator<T, boolean>|string, thisArg? : any) : T[];
+
+			// See https://github.com/Microsoft/TypeScript/blob/v1.8.10/lib/lib.core.es7.d.ts#L1122
```

Comment:
See the `map<U>(...): U[] `? I took the **U** solution/idea from https://github.com/Microsoft/TypeScript/blob/v1.8.10/lib/lib.core.es7.d.ts#L1122:

```JavaScript
map<U>(callbackfn: (value: T, index: number, array: T[]) => U, thisArg?: any): U[];
```

`ListIterator` from lodash being:
```JavaScript
interface ListIterator<T, TResult> {
  (value: T, index: number, collection: List<T>): TResult;
}
```
So the 'same' as `(value: T, index: number, array: T[]) => U` from lib.core.es7.d.ts

That's why I put a link to lib.core.es7.d.ts (and it's a good idea to link to lodash-3.10.d.ts#L7119 too).